person.rb

```diff
@@ -1,12 +1,16 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader :id, :parent_permission
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
     @id = Random.rand(1..100)
+    @rentals = []
+    super()
   end
 
   def can_use_services?
```

Comment:
- [ ] In your Person class, I noticed that you've correctly defined the rentals array to store the associated rentals. However, to fully meet the requirements, you need to implement the logic to associate Rental instances with Person objects. 👍 

- [ ] I recommend adding a method called `add_rental` in your Person class that allows you to associate rentals with a person just as I suggested earlier in the Book class 😉 
- By implementing this logic, you'll fully establish the `has-many relationship` between `Person` and `Rental`, fulfilling the requirements of the project 💯 
- Project Requirement

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
> Create the has-many side of Person and Rental (a person has many rentals).

Reply:
Well noted!💯
Thanks for the thorough and clear feedback 💪